axelor-project/src/main/java/com/axelor/apps/project/service/roadmap/SprintGeneratorServiceImpl.java

```diff
@@ -164,6 +168,29 @@ public Set<Sprint> generateSprints(
     return sprintSet;
   }
 
+  @Override
+  public String getSprintDomain(Project project) {
+    List<Sprint> sprintList = new ArrayList<>();
+    if (Objects.equals(
+        project.getSprintManagementSelect(), ProjectRepository.SPRINT_MANAGEMENT_PROJECT)) {
+      sprintList = project.getSprintList();
+    }
+    if (Objects.equals(
+        project.getSprintManagementSelect(), ProjectRepository.SPRINT_MANAGEMENT_VERSION)) {
+      List<List<Sprint>> sprintListList =
+          project.getRoadmapSet().stream()
+              .map(ProjectVersion::getSprintList)
+              .collect(Collectors.toList());
+      for (List<Sprint> sprints : sprintListList) sprintList.addAll(sprints);
+    }
+    if (sprintList.isEmpty()) {
+      return "self.id=0";
+    }
+    List<Long> ids = sprintList.stream().map(Sprint::getId).collect(Collectors.toList());
+
+    return "self.id in (" + Joiner.on(",").join(ids) + ")";
```

Comment:
StringHelper.getIdListString in axelor-utils can be used here, it manages the case where the list is empty.
```suggestion
    return String.format(
       "self.id in (%s)", StringHelper.getIdListString(sprintList))
```